draft-ietf-scitt-architecture.md

```diff
@@ -290,26 +298,26 @@ SCITT provides a standardized way to:
 
 ### Promotion of a Software Component by Multiple Entities
 
-A software component (e.g., a library) released by a certain original producer is becoming popular.
-The released software component is accompanied by a statement of authenticity (e.g., a detached signature).
+A software component (e.g., a library or software product) released by a legitimate producer is common practice for both open-source and commercial offerings.
+The released software component is accompanied by a statement of authenticity (e.g., an embedded or a detached signature).
 Over time, due to its enhanced applicability to various products, there has been an increasing amount of multiple providers of the same software component version on the internet.
 
-Some providers include this particular software component as part of their release package bundle and provide the package with proof of authenticity using their own issuer authority.
+Some providers include this particular software component as part of their release product/package bundle and provide the package with proof of authenticity using their issuer authority.
 Some packages include the original statement of authenticity, and some do not.
 Over time, some providers no longer offer the exact same software component source code but pre-compiled software component binaries.
 Some sources do not provide the exact same software component, but include patches and fixes produced by third-parties, as these emerge faster than solutions from the original producer.
 Due to complex distribution and promotion life-cycle scenarios, the original software component takes myriad forms.
 
 A consumer of a released software wants to:
 
-* understand if a particular provider is actually the original provider or a promoter
+* understand if a particular provider is actually the original authorized producer or an alternative party
 * know if and how the source, or resulting binary, of a promoted software component differs from the original software component
 * check the provenance and history of a software component's source back to its origin
-* assess whether to trust a promoter or not
+* assess whether to trust a component or product based on a downloaded package location and source supplier
 
 SCITT provides a standardized way to:
 
-* reliably discern if a provider is the original producer or is a trustworthy promoter or is an illegitimate provider
+* reliably discern if a provider is the original, trusted producer or is a trustworthy alternative provider or is an illegitimate provider
```

Comment:
```suggestion
* reliably discern if a provider is the original, legitimate producer or is a trustworthy alternative provider or is an illegitimate provider
```

Reply:
I can support this enhancement on line 320.

Reply:
Does using "legitimate" require the definition of what makes an issuer "legitimate"? 
"Trusted" is the choice of the verifier, or the TS when notarizing the signed statements. This seems more well-understood and less controversial.

Reply:
No objection to this enhancement.

Reply:
Based on the call 1/21/25, we've settled on "trusted" for line 301 and 320. 
Please reactivate if you feel strongly

Reply:
No objections to this consensus position. Good compromise. I presume that someone will make this change and I don't have to make this change - correct?

Reply:
Hi, @rjb4standards, yes the changes were committed to this PR. Please review: https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/328/files